www/manifest.json

```diff
@@ -1,19 +1,19 @@
-{

```

Comment:
this is very strange, i've only "moved" back all the files from /www/default to /www without changing them. furthermore no ones of the other files in the same directory looks changed